.github/workflows/pr_build.yml

```diff
@@ -91,6 +91,6 @@ jobs:
           retention-days: 1
 
   deploy:
-    uses: DeskproApps/app-template-vite/.github/workflows/subworkflow-deploy.yml@master
+    uses: DeskproApps/app-template-vite/.github/workflows/subworkflow-deploy.yml@SC-124017-refactor-apps-to-remove-iframe-resizer-dependency-and-make-iframe-content-sizing-more-robust
```

Comment:
perhaps it would be better to wait until it’s will be in the master branch and then update the app-template?

Reply:
If I do that, then this branch will build against the `master` version of `subworkflow-deploy.yml` which will fail the build as `actions/upload-artifact@v4` in this file is not compatible with `actions/download-artifact@v3` in `master`, so I have to point it to this branch where it is `actions/download-artifact@v4`. When it's in `master, I'll create another PR to change it back to `master` so it's only temporary to get the builds to pass.

Reply:
To be clear, as soon as this PR is merged, I'll open another straight after to correct the namings.